theme/parts/tabsbar.css

```diff
@@ -510,3 +510,84 @@ tab {
 		margin-right: -16px;
 	}
 }
+
+/* OPTIONAL: Use tabs as headerbar */
+@supports -moz-bool-pref("gnomeTheme.tabsAsHeaderbar") {
+	/* Rearrange bars */
+	#navigator-toolbox #titlebar {
+		-moz-box-ordinal-group: 0;
+	}
+	#navigator-toolbox #titlebar #TabsToolbar {
+		-moz-box-ordinal-group: 0;
+	}
+	#navigator-toolbox #titlebar #toolbar-menubar {
+		-moz-box-ordinal-group: 1;
+	}
+	#navigator-toolbox #nav-bar {
+		-moz-box-ordinal-group: 1;
+	}
+	#navigator-toolbox #PersonalToolbar {
+		-moz-box-ordinal-group: 2;
+	}
+	
+	/* Remove nav-bar rounding and padding */
+	:root[tabsintitlebar]:not([inFullscreen]) #nav-bar,
+	:root[tabsintitlebar][inFullscreen] #nav-bar {
+		border-radius: 0 !important;
```

Comment:
You will need a higher  or equal selector than `:root[tabsintitlebar][sizemode="normal"]:not([gtktiledwindow="true"]) #nav-bar`, otherwise I'm getting the navbar rounded when the window is unmaximized:
![Captura desde 2022-10-22 23-30-48](https://user-images.githubusercontent.com/6210397/197373951-b4f916f7-3575-4381-bc11-d9ff9325243d.png)

Reply:
I'm struggling to see it even in your picture, but it should be fixed now. Looking at it again, I'm not sure why I had two selectors for... both fullscreen and not fullscreen? Must've been some artifact over time.